### PR TITLE
Request to merge Feature navigation

### PR DESCRIPTION
**Pull request by Jmagero**

### CODE REVIEW

**Comment by Jmagero** [comment]
- [x] ModifyModify the Awesome books application to have:
- A Navigation bar.Three content sections:
- Books list.
- Add book form.
- Contact info
- [x] Make sure that when you click on any of the links, the main area changes to display only one of the three content sections
- [x] Display the current date below the navigation bar

.

File: index.html [hunk 1]
```diff
@@ -5,21 +5,44 @@
...
+        <li>Our email: mail@mail.com</li>
+        <li>Our phone number: 900130012021</li>
+        <li>Our address: Palm Beach 78,8451 City, Country</li>
+      </ul>
+    </section>
+    <footer>
+      Copyright
+    </footer>
+  </main>
+  <script type="module" src="/assets/main.js"></script>
```
Comment (teekaytech) [inline index.html]: - [ ] To ensure that your website follows the wireframe specifications, kindly make the following changes:
- Style your navbar items in such a way that they're aligned to the right side of the page
- Style your footer to be at the bottom of the page: maybe that's one of the reasons why they named it **foot**er :smile: 

**Given Wireframe**
awesome_books_full_website

**Your Design**
project

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

**Your website looks pretty cool :wink:**
_Your PR title, summary, and commit messages look good :+1: 
Your linters are running with no errors :100:
The existing features still work :ok_hand:_

__However, you need to make some required changes before moving on to the next assignment.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes** :recycle: 
Check the comments under the review.

### **Optional Changes** :zap: 
_Every comment with the **OPTIONAL** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

**Comment by kimothokamau** [comment]
- Fix nav links spacing
- Fix footer positioning

**Review by teekaytech — APPROVED** [review]
## Status: Approved :green_circle: 

### Congratulations team :tada: :clinking_glasses: 

**Your project is complete!  This is a convenient point to merge :rocket:**

----- 
**Other things to take care of before merging**
- Update your Readme file:
  - The `issues page` link is directing the user to another project entirely. Kindly fix this.
  - You may consider adding the project screenshot too.

- - - - - -

**_Feel free to leave any questions or comments in the PR thread if something is not 100% clear._**

Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
_Cheers,_
**[Taofeek Olalere](https://github.com/teekaytech)**
__Happy Coding :smiley:__

**Pull request merged**